client/my-sites/checkout/checkout-thank-you/rebrand-cities-thank-you.jsx

```diff
@@ -2,30 +2,41 @@
  * External dependencies
  */
 import React, { Component } from 'react';
-import Gridicon from 'gridicons';
 import { localize } from 'i18n-calypso';
+import ThankYouCard from 'components/thank-you-card';
 
 class RebrandCitiesThankYou extends Component {
+	renderLogo() {
+		return <img src="/calypso/images/rebrand-cities/wp-rebrand-cities-logo.svg" />;
+	}
+
 	render() {
 		const { translate } = this.props;
 
 		return (
-			<div className="checkout-thank-you__header">
-				<div className="checkout-thank-you__header-content">
-					<span className="checkout-thank-you__header-icon">
-						<Gridicon icon="trophy" size={ 72 } />
-					</span>
-
-					<div className="checkout-thank-you__header-copy">
-						<h1 className="checkout-thank-you__header-heading">
-							{ translate( 'Thank you for signing up!' ) }
-						</h1>
-
-						<h2 className="checkout-thank-you__header-text">
-							{ translate( 'We will contact you shortly' ) }
-						</h2>
-					</div>
-				</div>
+			<div className="plan-thank-you-card checkout-thank-you__rebrand-cities">
+				<ThankYouCard
+					name={ translate( 'Rebrand Cities package' ) }
+					price="$299.00"
```

Comment:
Should we be passing this value dynamically vs hard coding it? Can we show the discounted price if that's what they paid?